### ISSUE TITLE
Head of Ansible stops executing after first include

### ISSUE THREAD

**Issue by ghost** [ansible/ansible#19875]
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
Playbook includes

##### ANSIBLE VERSION
```
ansible 2.3.0
  config file =
  configured module search path = Default w/o overrides
```

##### CONFIGURATION
Nothing special 

##### OS / ENVIRONMENT
Ubuntu 16,  
Linux ip-172-30-17-159 4.4.0-45-generic #66-Ubuntu SMP Wed Oct 19 14:12:37 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux

##### SUMMARY
Playbook exits with warning after first include 

##### STEPS TO REPRODUCE
```ansible-playbook -i 'localhost' -c local /vagrant/ansible/playbook.yml -b -vvvv```
With tasks/main.yaml 

```
---

- include: ntp.yml
- include: limits.yml
- include: javaversion.yml
- include: phantomjs.yml
- include: postgres.yml
.
.
.
- name: Copy jmx security files to looker home
  copy: src={{ item }} dest={{ jmx_security_dir }}/{{ item }} owner={{ lookerops }} group={{ lookeropsgr }} mode=0400
  with_items:
    - jmxremote.access
    - jmxremote.password
    
- name: Copy looker startup script to looker home
  copy: src=looker dest=/home/looker/looker/looker owner=looker group=looker mode=744

- name: Copy looker startup script overrides to looker home
  copy: src=lookerstart.cfg dest=/home/looker/looker/lookerstart.cfg owner=looker group=looker mode=744

- name: Copy looker systemd file
  copy: src=looker.service dest=/etc/systemd/system/looker.service owner=root group=root mode=644

- name: Start Looker service
  systemd: name=looker state=started enabled=yes

- include: cron.yml

- meta: flush_handlers
```
##### EXPECTED RESULTS
It should execute all of the includes. 

##### ACTUAL RESULTS
Executes first one only. Confirmed it's the order by commenting and reordering, and it's always the first one that's run, not a specific include. 

```
.
.
.
ubuntu@ip-172-30-17-159:~$ ansible-playbook -i 'localhost' -c local /vagrant/ansible/playbook.yml -b
 [WARNING]: Host file not found: localhost

 [WARNING]: provided hosts list is empty, only localhost is available


PLAY [localhost] ****************************************************************************************

TASK [Gathering Facts] **********************************************************************************
ok: [localhost]

TASK [looker : Ensure NTP is installed] *****************************************************************
ok: [localhost]

TASK [looker : yum] *************************************************************************************
skipping: [localhost]

TASK [looker : Ensure NTPD is running] ******************************************************************
ok: [localhost]

TASK [looker : service] *********************************************************************************
skipping: [localhost]

PLAY RECAP **********************************************************************************************
localhost                  : ok=3    changed=0    unreachable=0    failed=0

PLAY RECAP **********************************************************************************************
localhost                  : ok=3    changed=0    unreachable=0    failed=0

```
Confirmed this works on 2.2, however I wanted to use some 2.3 features.

**Comment by jimi-c** [comment]
This should be fixed by ac00c8a6601ddb34737ccbffc14cdd1c10359779.

If you continue seeing any problems related to this issue, or if you have any further questions, please let us know by stopping by one of the two mailing lists, as appropriate:

   * https://groups.google.com/forum/#!forum/ansible-project - for user questions, tips, and tricks
   * https://groups.google.com/forum/#!forum/ansible-devel - for strategy, future planning, and questions about writing code

Because this project is very active, we're unlikely to see comments made on closed tickets, but the mailing list is a great way to ask questions, or post if you don't think this particular issue is resolved.

Thank you!

**Comment by ghost** [comment]
Yep, confirmed works. Just checked it out minutes before you checked in the fix! Cheers.
